make_common/bob_robotics.mk

```diff
@@ -146,3 +146,11 @@ ifdef WITH_SFML_GRAPHICS
         CXXFLAGS += `pkg-config --cflags sfml-graphics`
         LINK_FLAGS += `pkg-config --libs sfml-graphics`
 endif
+
+ifdef WITH_SPINEML_SIMULATION
+    ifndef GENN_PATH
+        $(error Environment variable GENN_PATH must be defined)
+    endif
+    CXXFLAGS += -I$(GENN_PATH)/include/spineml/simulator
+    LINK_FLAGS += -L$(GENN_PATH)/lib -lspineml_simulator -lspineml_common -ldl
+endif
```

Comment:
Are we using GeNN v4 or v3 here? If we're using v4 then I guess we don't need ``GENN_PATH`` set, right?

Reply:
GeNN 4 so this isn't otherwise required - I just couldn't think of a better way of locating the GeNN lib directory.

Reply:
But if the user has installed GeNN with make install then shouldn't it be somewhere standard? If you just include spineml/simulator/whatever.h then you shouldn't need to put in a full path (I also feel that all the GeNN-related headers should probably live under /usr/include/genn or something so no extra include paths would be needed)

Reply:
That is optional though - personally I have it in  ~/genn for easy meddling :smile:

Reply:
Although that does mean that even if you run ``make install`` you should still set ``GENN_PATH``. I feel like that should be optional, but it's no biggie as long as the docs tell you to do that.

Reply:
Through the powers of bash I've come up with a much nicer solution - ``genn-buildmodel.sh`` should always be in your path and should always live in ``$GENN_PATH/bin`` therefore you can easily figure out ``GENN_PATH`` (if you've done ``make install`` it'll be ``/usr/local`` or similar)

Reply:
Good thinking.